go.mod

```diff
@@ -1,6 +1,6 @@
-module decred.org/dcrwallet/v4
+module decred.org/dcrwallet/v5
 
-go 1.20
+go 1.22
```

Comment:
this can be debated for the ages but this is the minimum version we do support and now is as good of a time as any to bump it.

Reply:
https://protobuf.dev/reference/go/go-generated/#package

> The Go import path is locally specified in a .proto file by declaring a go_package option

> the package name will be derived by default from the import path in a reasonable manner.

> The import path is used to determine which import statements must be generated when one .proto file imports another .proto file. 

Because dcrwallet doesnt have any proto files importing other proto files, it seems like the whole path is not used. Only the package name (the bit after the final slash) is used.

Verified this locally by setting `option go_package = "bullshit/nonsense/walletrpc"` and running regen.sh, nothing changes. `option go_package = "bullshit/nonsense/walletrpc2"` updates the package name of the generated files to "walletrpc2"